CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v2.1.1 / 2021-07-28
+
+* [CHANGE] Replace multiListWatcher with independent listWatchers per namespace #1499
```

Comment:
I think this should be a bugfix since it fixes the permission issue from #1413. Also, we could reuse the release note that @fpetkovski added to his PR:
> Fixes a bug where KSM did not export any metrics when it had no permissions for resources in at least one namespace